Case/src/StorageSystem.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PluckList.src
+{
+    public class StorageSystem
+    {
+        public List<Item> Items { get; }
+        public StorageSystem()
+        {
+            Items = new List<Item>();
+        }
+        public void SetItems(PluckList pluckList)
+        {
+            foreach (Item pluckItem in pluckList.Lines)
+            {
+                bool notInList = true;
+
+                foreach (Item storageItem in Items)
+                {
+                    if (pluckItem.ProductID == storageItem.ProductID)
+                    {
+                        notInList = false;
+                        break;
+                    }
+                }
```

Comment:
Can use Items.All(...).
```suggestion
                bool notInList = Items.All(storageItem => pluckItem.ProductID != storageItem.ProductID);
```

Reply:
Replaced by sortedItems.DistinctBy()